server/container/util/src/test/resources/docker/spamassassin/README.md

```diff
@@ -0,0 +1,44 @@
+docker-spamassassin
```

Comment:
is the licence of these files compatible with the Apache one?

Reply:
no need of this file